README.md

```diff
@@ -11,6 +11,8 @@ Microservice to allow 3rd party discovery services to determine the availability
 
 ## Additional information
 
+The expectation is that the **mode of issuance "serial"** and **nature of content "journal" and "newspaper"** are present in the target system. The module uses these values to qualify an instance as a periodical.
```

Comment:
I guess this is fine for now, but I am concerned that these values are taken from controlled vocabulary and that an institution may want to translate these values to another language. These mappings should probably be externalized, settable perhaps via an RTAC settings app in FOLIO.

Reply:
This documentation was added as it was discussed with @craigmcnally